controllers/idler/idler_controller.go

```diff
@@ -71,8 +55,9 @@ type Reconciler struct {
 // Note:
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
-func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	logger := r.Log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
+func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.Result, error) {
+	logger := log.FromContext(ctx)
+	//logger := r.Log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
```

Comment:
this line can be removed

Reply:
good catch - that slipped out of my notice - let me go over the code again and check if there are more of these

Reply:
removed 114a7b9 :ghost: